woocommerce/woocommerce-gutenberg-products-block
# Remove colors from icons displayed in the shortcuts inserter

As discussed in https://github.com/WordPress/gutenberg/pull/17237, we would like to display block icons without color in the shortcuts inserter.

### Screenshots

![image](https://user-images.githubusercontent.com/3616980/64436125-02c2d080-d0c4-11e9-8791-afee06a60fc0.png)

### How to test the changes in this Pull Request:

1. Make it so _Product Search_, _All Reviews_, _Reviews by Product_ and _Reviews by Category_ appear in the inline inserter (the one that appears at the bottom right of the editor when you add a new line). Doing that is a bit annoying because you have to add the block several times in a post until it appears in the shortcuts section, but I couldn't find any easier way.
2. Verify it doesn't have WooCommerce color but instead it's gray.

### Changelog

> All block icons are displayed gray in the editor shortcuts inserter.
